src/eval/execution.rs

```diff
@@ -14,15 +14,43 @@ static INIT_V8: Once = Once::new();
 
 pub fn evaluate(nix_expr: &str) -> EvalResult {
     initialize_v8();
+    // Declare the V8 execution context
     let isolate = &mut v8::Isolate::new(Default::default());
     let scope = &mut v8::HandleScope::new(isolate);
     let context = v8::Context::new(scope);
     let scope = &mut v8::ContextScope::new(scope, context);
+    let global = context.global(scope);
 
+    let import_module_attr = v8::String::new(scope, "importNixModule").unwrap();
+    let import_module_fn = v8::Function::new(scope, import_nix_module).unwrap();
+    global
+        .set(scope, import_module_attr.into(), import_module_fn.into())
+        .unwrap();
+
+    // Execute the Nix runtime JS module, get its exports
+    let nixjs_rt_str = include_str!("../../nixjs-rt/dist/lib.mjs");
+    let nixjs_rt_obj = exec_module(nixjs_rt_str, scope).unwrap();
+
+    // Set them to a global variable
+    let nixrt_attr = v8::String::new(scope, "n").unwrap();
+    global
+        .set(scope, nixrt_attr.into(), nixjs_rt_obj.into())
+        .unwrap();
+
+    let root_nix_fn = eval_nix_fn_from_string(scope, nix_expr)?;
+
+    nix_value_from_module(scope, root_nix_fn, nixjs_rt_obj)
+}
+
+fn eval_nix_fn_from_string<'s>(
```

Comment:
Nitpick: maybe a slightly different name for the function:
```suggestion
fn nix_expr_to_js_function<'s>(
```